src/Microsoft.OData.Core/UriParser/Binders/EndPathBinder.cs

```diff
@@ -111,7 +111,7 @@ internal static SingleValueNode CreateParentFromImplicitRangeVariable(BindingSta
         internal SingleValueOpenPropertyAccessNode GeneratePropertyAccessQueryForOpenType(EndPathToken endPathToken, SingleValueNode parentNode)
         {
             if (parentNode.TypeReference == null ||
-                parentNode.TypeReference.Definition.IsOpenType() ||

```

Comment:
EdmLib has `IsOpen()` extension method defined ,  `IsOpenType()` was similar function defined in ODL and I removed it.